utils/active_timer.js

```diff
@@ -0,0 +1,61 @@
+const cron = require("node-cron");
+
+var TIMER_POOLS = {};
+
+module.exports = {
+   start: (req, element) => {
+      if (element == null || !element.isEnabled) return;
+
+      let tenantID = req.tenantID();
+      if (TIMER_POOLS[tenantID] == null) {
+         TIMER_POOLS[tenantID] = {};
+      }
+
+      // Stop
+      if (TIMER_POOLS[tenantID][element.id] != null) {
+         this.stop(req, element.id);
+      }
+
+      let cronExpression = element.getCronExpression();
+      req.log(`::: START CRON job [${cronExpression}] - ${element.id}`);
+
+      TIMER_POOLS[tenantID][element.id] = cron.schedule(cronExpression, () => {
```

Comment:
does `cron.schedule()` have the possibility of failing?  If so, we should check that.

Reply:
I walked through the code. It does not pass any `.error` object.
It pass the current DateTime at the first parameter.
https://github.com/node-cron/node-cron/blob/master/src/node-cron.js#L29
https://github.com/node-cron/node-cron/blob/master/src/task.js#L17
https://github.com/node-cron/node-cron/blob/master/src/scheduler.js#L30

I think we might need to catch the fail from `try catch` statement.